src/main/resources/db/migration/V7__addCityColumnToEventTable.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE EVENT
+ADD COLUMN CITY VARCHAR NOT NULL;
```

Comment:
O que acontece com eventos já existentes?

Reply:
Como eu não tinha nada no banco ainda, acabei colocando essa alteração, mas nesse caso se eu tivesse algum evento eles teriam a coluna city vazia

Reply:
Acho que daria problema se houvessem registro na base já. Quando usamos esse esquema de migrações, sempre precisamos analisar e nos preocupar com mudanças destrutivas e/ou que impões novas restrições, como o caso do `NOT NULL`...